cmd/ntrip-mqtt-gateway/main.go

```diff
@@ -6,6 +6,13 @@ import (
 	"github.com/spf13/viper"
 )
 
+// Currently using Viper to parse the config file directly into a Gateway object.
+// Could instead have an intermediary Config object for more flexibility in adding
```

Comment:
i guess if the Config object is simple and readable enough, you wouldn't really need a json file. you could also generate the json dynamically, but maybe that's just more complicated